docs/research/ticket-selection-algorithm.md

```diff
@@ -0,0 +1,8 @@
+# <img class="dcr-icon" src="/img/dcr-icons/QuestionTicket.svg" /> Ticket Selection Algorithm
+
+Every block, 5 tickets are pseudorandomly selected from the ticket pool to [vote](../../proof-of-stake/overview/). Below is a high-level description of the ticket selection algorithm. A more detailed, low-level walkthrough of the ticket selection code can be found in this [video](https://www.youtube.com/watch?v=eysGWVhDFWY).
```

Comment:
link should be to a `.md` file

Reply:
Fixed. FYI, I do remember trying `.md` before and it wasn't working, so went with the `/`. I think my mistake was that with the `/`, I was able to use a relative path that wasn't technically correct (`../../proof-of-stake/overview/`) and it worked. Using `.md` with that path doesn't work. If you use the correct relative path (`../proof-of-stake/overview.md`) it does.

Reply:
Exactly its a relative path from the current source file to target source file. This looks good now.